english/watch-alien/watch-alien.md

```diff
@@ -5,3 +5,5 @@ With no idea what's going on, and your heart rate already slightly elevated, you
 to panic.  You begin to smell a faint odor that you don't recognize.
 
 This time, it's not the dog.
+
+A viscous, dark green liquid starts crawling out from beneath the door. You quickly tuck your knees up and crouch on your sofa. The green fluid starts to glow, an unerving regular beat - almost like - no, it couldn't be - you realise it is your heartbeat and start to panic. Why is it the same?
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.